### DISCUSSION TITLE
getting_started/introduction/introduction_to_godot

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#39] [User-contributed notes]
# getting_started/introduction/introduction_to_godot

This article is here to help you figure out whether Godot might be a good fit for you. We will introduce some broad features of the engine to give you a feel for what you can achieve with it and an...

https://docs.godotengine.org/en/stable/getting_started/introduction/introduction_to_godot.html

**Comment by Itx-Psycho0** [comment]
For complete beginners, the official Getting Started step-by-step guide pairs well with this introduction and helps translate concepts into practice:
https://docs.godotengine.org/en/stable/getting_started/step_by_step/index.html
For visual learners, GDQuest’s beginner tutorials provide clear explanations of core concepts like scenes, nodes, and GDScript fundamentals:
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html
Suggestion:
New users often benefit from building a very small 2D project first (even if their goal is 3D) to understand the scene tree, nodes, and scripting workflow before moving on to more complex features.

**Reply by JRSBoggs** [reply]
It looks like the GDScript Fundamentals link leads to a 404 now.

**Reply by Itx-Psycho0** [reply]
Thanks for the heads up, I fixed the GDScript fundamental link.
Happy learning.
